packages/hd/src/mnemonic/index.ts

```diff
@@ -50,37 +55,21 @@ function salt(password = ""): string {
 export function mnemonicToSeedSync(mnemonic = "", password = ""): Buffer {
   const mnemonicBuffer = Buffer.from(mnemonic.normalize("NFKD"), "utf8");
   const saltBuffer = Buffer.from(salt(password.normalize("NFKD")), "utf8");
-  return pbkdf2Sync(
-    mnemonicBuffer,
-    saltBuffer,
-    PBKDF2_ROUNDS,
-    KEY_LEN,
-    "sha512"
+  return Buffer.from(
+    pbkdf2(sha512, mnemonicBuffer, saltBuffer, {
+      c: PBKDF2_ROUNDS,
+      dkLen: KEY_LEN,
+    })
   );
 }
 
 export function mnemonicToSeed(mnemonic = "", password = ""): Promise<Buffer> {
-  return new Promise((resolve, reject) => {
-    try {
-      const mnemonicBuffer = Buffer.from(mnemonic.normalize("NFKD"), "utf8");
-      const saltBuffer = Buffer.from(salt(password.normalize("NFKD")), "utf8");
-      pbkdf2(
-        mnemonicBuffer,
-        saltBuffer,
-        PBKDF2_ROUNDS,
-        KEY_LEN,
-        "sha512",
-        (err, data) => {
-          if (err) {
-            reject(err);
-          }
-          resolve(data);
-        }
-      );
-    } catch (error) {
-      reject(error);
-    }
-  });
+  const mnemonicBuffer = Buffer.from(mnemonic.normalize("NFKD"), "utf8");
+  const saltBuffer = Buffer.from(salt(password.normalize("NFKD")), "utf8");
+  return pbkdf2Async(sha512, mnemonicBuffer, saltBuffer, {
+    c: PBKDF2_ROUNDS,
+    dkLen: KEY_LEN,
+  }).then(Buffer.from);
```

Comment:
these lines aren't covered. could you please add tests for them? 

vector and cases can be found in the bip39 repo https://github.com/bitcoinjs/bip39/blob/4fce199e73f7ab68bdf9f7382dc10f7e1adaa499/test/index.js

Reply:
Both `mnemonicToSeed` and `mnemonicToSeedSync` are covered by tests.
https://github.com/ckb-js/lumos/blob/develop/packages/hd/tests/mnemonic/index.test.ts#L42